app/pb_kits/playbook/pb_label_pill/_label_pill.scss

```diff
@@ -0,0 +1,8 @@
+.pb_label_pill {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  .pb_caption {
```

Comment:
Can you add a unique class to pb_caption and reference it here? Add a classname in your kit class file.

Reply:
Added "pb_label_pill_label" class name